api/src/services/notifications.test.ts

```diff
@@ -199,7 +200,7 @@ describe('Integration Tests', () => {
 					template: {
 						name: 'base',
 						data: {
-							url: `/admin/users/${userDetail.id}`,
+							url: `${env.PUBLIC_URL}/admin/users/${userDetail.id}`,
```

Comment:
This was due to the env mock being unintentionally removed in https://github.com/directus/directus/pull/16462/files#diff-20e1f0d2df91af9fadda290866eb9522622fa12acd4a73a1f294e6b5e67aaa59, but IIRC it was re-added afterwards so not sure why this mock is not here again. Pushed a commit to mock the env again 👍